src/src/fileLoger/hillel/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package fileLoger.hillel;
+
+class FileLoggerConfiguration {
+    private String file;
+    private LoggingLevel level;
+    private long maxFileSize;
+    private String format;
+
+    public FileLoggerConfiguration(String file, LoggingLevel level, long maxFileSize, String format) {
+        this.file = file;
```

Comment:
It would be better to add validation for all configuration parameters. In this case, it's ok to apply null-checking to the reference types and check whether `maxFileSize` has a correct value, e.g. at least 1 byte